Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -574,5 +538,47 @@ private void Grid_Loaded(object sender, RoutedEventArgs e)
             var itemContainer = item as GridViewItem;
             itemContainer.ContextFlyout = ItemContextMenuFlyout;
         }
+
```

Comment:
I prefer this implementation because we can unload thumbnails from the visual tree. We could also remove the byte[] array during unload but it would be more intensive to call Windows.Storage again for the same image once reloaded.